content/en/docs/tasks/inject-data-application/define-environment-variable-container.md

```diff
@@ -80,6 +80,49 @@ Pod:
    DEMO_FAREWELL=Such a sweet sorrow
    ```
 
+### Setting container environment variables with `envFrom`
+
+In this exercise, you create a Pod that runs one container.
+But first we need to reference it to a ConfigMap 
+
+{{< codenew file="pods/inject/configmap-envFrom.yaml" >}}
+
+Now as the kind: ConfigMap is created we need to 
+reference it to a pod. 
+
+{{< codenew file="pods/inject/envFrom.yaml" >}}
+
+You can also use Secret instead of ConfigMap for the configuration
+
+```
```

Comment:
nit: could append `yaml` here for syntax highlighting.

Reply:
done